### PR TITLE
axios thunk dumpster fire :eyes:

### PR DESCRIPTION
**Pull request by beetai**
naniiyeeeEEEEEEEEEEEEEEEEEEEEEEEEEEEeeeeet, 4 u

### CODE REVIEW

File: server/routes/users.js [hunk 1]
```diff
@@ -12,15 +12,16 @@ let users = [
     id: '1',
     username: 'testusername',
     firstname: 'madman',
-    lastname: 'naniiyeeet'
+    lastname: 'naniiyeeeEEEEEEEEEEEEEEEEEEEEEEEEEEEeeeeet'
```
Comment (yqj2k) [inline server/routes/users.js]: :eyes:

Comment (DSep) [inline server/routes/users.js]: DELYEET.

**Review by DSep — CHANGES_REQUESTED** [review]
Fix this, and the rest is good.

File: server/routes/users.js (hunk 1 above)
Comment (baconandchips) [reply server/routes/users.js]: 🔥

**Review by baconandchips — APPROVED** [review]

**Review by DSep — APPROVED** [review]

**Pull request merged**